monetio/sat/_modis_l2_mm.py

```diff
@@ -20,18 +21,23 @@ def read_dataset(fname, variable_dict):
     """
     from monetio.sat.hdfio import hdf_close, hdf_list, hdf_open, hdf_read
 
+    epoch_1993 = int(datetime(1993, 1, 1, tzinfo=timezone.utc).timestamp())
+
     print("reading " + fname)
 
     ds = xr.Dataset()
 
     f = hdf_open(fname)
     hdf_list(f)
-    latitude = hdf_read(f, "Latitude")  # noqa: F841
-    longitude = hdf_read(f, "Longitude")  # noqa: F841
-    start_time = hdf_read(f, "Scan_Start_Time")  # noqa: F841
+    # Geolocation and Time Parameters
+    latitude = hdf_read(f, "Latitude")
+    longitude = hdf_read(f, "Longitude")
+    # convert seconds since 1993 to since 1970
+    start_time = hdf_read(f, "Scan_Start_Time") + epoch_1993
     for varname in variable_dict:
         print(varname)
         values = hdf_read(f, varname)
+        print('min, max: ', values.min(), ' ', values.max())
```

Comment:
please the formatter
```suggestion
        print("min, max: ", values.min(), " ", values.max())
```